msvc2017/Common/Common.vcxproj

```diff
@@ -27,19 +31,25 @@
     <CharacterSet>Unicode</CharacterSet>
     <PlatformToolset>v141</PlatformToolset>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
+    <ConfigurationType>StaticLibrary</ConfigurationType>
+    <UseDebugLibraries>true</UseDebugLibraries>
+    <CharacterSet>Unicode</CharacterSet>
+    <PlatformToolset>v141</PlatformToolset>
```

Comment:
Is this event the correct platform toolset?

Reply:
Not sure what you mean... v141 is the MSVC 2017 compiler toolset. These are MSVC 2017 project files.

Reply:
Isn't there a distinction between a 64 bit toolset and a 32 bit toolset?

Reply:
Toolset (compiler version) and platform (x64 or Win32) are independent settings in the project configuration interface.